src/main/java/com/cricket/project/controller/MatchController.java

```diff
@@ -0,0 +1,40 @@
+package com.cricket.project.controller;
+
+import com.cricket.project.dto.MatchDto;
+import com.cricket.project.model.InningStats;
+import com.cricket.project.model.Match;
+import com.cricket.project.model.MatchScoreCard;
+import com.cricket.project.model.PlayerScoreCard;
+import com.cricket.project.service.InningService;
+import com.cricket.project.service.ScoreCardService;
+import com.cricket.project.service.impl.MatchServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+@RestController
+@Slf4j
+public class MatchController {
+    @Autowired
+    private MatchServiceImpl matchService;
+
+    @Autowired
+    private ScoreCardService scoreCardService;
+    @PostMapping("/startMatch")
+    public Match startMatch(@RequestBody MatchDto document) {
+         return matchService.setUpMatch(document);
+    }
+
+    @GetMapping("/scoreCardForAMatch")
+    public MatchScoreCard PlayerScoreCardForAMatch(
```

Comment:
rename this method properly with the proper naming conventions https://www.educative.io/answers/what-are-the-naming-conventions-in-java

Reply:
also this providing the match score card and not the player score card

Reply:
> also this providing the match score card and not the player score card

yes observed it and made changes